academic-services/knowledge-exploration-service/tutorial-schema-design.md

```diff
@@ -96,45 +99,45 @@ The design goals above guide us to define a [schema for the linked sample librar
 
 | Attribute Name | Description| Index Data Type | Index Operations |
```

Comment:
>Description [](start = 19, length = 11)

Your attribute descriptions here are describing the role of the attribute in the application, not what the attribute actually *is*. This is inconsistent with how you describe attributes in the next "search" tutorial.

Can you make these descriptions align with how you describe the attributes in the "search" tutorial?